tdrs-backend/tdpservice/conftest.py

```diff
@@ -243,3 +245,8 @@ def test_private_key():
         key_size=4096,
     )
     yield get_private_key(key)
+
+
+# Register factories with pytest-factoryboy for automatic dependency injection
+# of model-related fixtures into tests.
+register(OwaspZapScanFactory)
```

Comment:
This instructs pytest-factoryboy to create [Factory and Model fixtures](https://pytest-factoryboy.readthedocs.io/en/latest/#factory-fixture) which we can reuse in varying ways.